azure-pipelines.yml

```diff
@@ -86,6 +86,7 @@ stages:
   jobs:
   - job:
     pool: sonictest
+    condition: ne(stageDependencies.BuildVS.vs.outputs['SetVar.SKIP_VSTEST'], 'YES')
```

Comment:
There is another condition is VSBuild succeeded, right?
What is the behavior, if the VSBuild Failed, but the SKIP_VSTEST is set.

Can we set the condition in Stage level, not for each of the test job?

Reply:
yes, it needs succeeded()
We can't set in stage level. it don't work. https://dev.azure.com/mssonic/build/_build/results?buildId=108401&view=results